### ISSUE TITLE
Incompatible with Microsoft.EntityFrameworkCore.Sqlite nuget

### ISSUE THREAD

**Issue by Anil86** [raisedapp/Hangfire.Storage.SQLite#30]
I'm using SQLite as the database for my app. So I installed `Microsoft.EntityFrameworkCore.Sqlite` *version 3.1.7* nuget & ran the project (Blazor Server App) which caused following exception:

```console
System.MissingMethodException: Method not found: 'System.String SQLitePCL.raw.sqlite3_column_name(SQLitePCL.sqlite3_stmt, Int32)'.
```

> Without `Microsoft.EntityFrameworkCore.Sqlite`, Hangfire works perfectly.

**Comment by felixclase** [comment]
I'll be checking

**Comment by AutumnEvans418** [comment]
I also have this issue with `Microsoft.EntityFrameworkcore.sqlite` version 3.1.3.

**Comment by AutumnEvans418** [comment]
I have discovered the solution.  Adding `<PackageReference Include="sqlite-net-pcl" Version="1.7.335" />` to my csproj solved the issue.  There was just a version mismatch

**Comment by Yaevh** [comment]
Same problem, same resolution here. The bug would be easily fixed by updating `sqlite-net-pcl` to 1.7.335 in the library (`<PackageReference Include="sqlite-net-pcl" Version="1.7.335" />`), but it would require dumping .NET Framework 4.5 version, since 1.7.335 doesn't support 4.5 :/

**Comment by lulzzz** [comment]
same issue. please update.

**Comment by felixclase** [comment]
> Same problem, same resolution here. The bug would be easily fixed by updating `sqlite-net-pcl` to 1.7.335 in the library (`<PackageReference Include="sqlite-net-pcl" Version="1.7.335" />`), but it would require dumping .NET Framework 4.5 version, since 1.7.335 doesn't support 4.5 :/

![image](https://user-images.githubusercontent.com/1153361/107866344-23a95c80-6e46-11eb-8e68-71446924f38f.png)

So as not to break compatibility with those who are still using .net 4.5
